File: Python Word Examples/TablesApiDemo/Table/getTableFormat.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.TablesApi(configuration)

file_name = 'Table.docx'
index = 0
node_path = 'Section/0/Body/0'
result = api.get_table_format(file_name, node_path, index, folder="input")

File: PHP Word Examples/TablesApiDemo/Table/getTableFormat.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\TablesApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new TablesApi($configuration);

$fileName = "GetTableFormat_1.docx";
$node_path = 'Section/0/Body/0';
$index = 1;
$folder = 'input';
$storage = null;
$password = null;
$result = $apiInstance->getTableFormat($fileName,$node_path, $index,$folder,$storage,$password);